packages/components/src/templates/next/components/internal/ContentPageHeader/ContentPageHeader.tsx

```diff
@@ -14,34 +13,26 @@ const ContentPageHeader = ({
   LinkComponent,
 }: ContentPageHeaderProps) => {
   return (
-    <div className="bg-site-secondary-100">
-      <div className="mx-auto max-w-container">
-        <div className="flex max-w-[880px] flex-col gap-8 px-6 py-8 md:px-10 lg:gap-12 lg:py-16">
-          <div className="hidden lg:block">
-            <Breadcrumb
-              links={breadcrumb.links}
-              LinkComponent={LinkComponent}
-            />
-          </div>
-          <div className="flex flex-col">
-            <h1 className="text-[2.75rem] font-semibold leading-tight text-site-secondary lg:text-[3.75rem]">
-              {title}
-            </h1>
-            <div className="pt-6 lg:pb-2">{`Last updated ${getFormattedDate(lastUpdated)}`}</div>
-            <BaseParagraph
-              content={summary}
-              className="text-content text-paragraph-01"
-            />
+    <div className="bg-site-secondary-100 text-base-content-strong">
+      <div className="mx-auto flex max-w-screen-xl flex-col gap-8 px-6 py-8 md:px-10">
+        <div className="flex flex-col">
+          <Breadcrumb links={breadcrumb.links} LinkComponent={LinkComponent} />
+          <div className="mt-8 flex flex-col gap-5 md:mt-6">
+            <h1 className="prose-display-lg">{title}</h1>
+            <p className="prose-title-lg-regular">{summary}</p>
```

Comment:
nit: unrelated to this PR but I think a line height of 1.3 here might be too small to read, especially given our font it makes the text seem quite close together